backend/routes/locations.py

```
@@ -4,21 +4,114 @@
+                result = {
+                    'name': name,
+                    'displayName': loc.address,
+                    'latitude': loc.latitude,
+                    'longitude': loc.longitude,
+                    'country': country,
+                    'state': state,
+                    'timezone': tz
+                }
+                results.append(result)
+                
+            except Exception as e:
```

Ruff (0.11.9)

Local variable `e` is assigned to but never used

Remove assignment to unused variable `e`

(F841)